Sources/Apodini/Properties/State.swift

```diff
@@ -56,12 +56,13 @@ extension State where Element: ExpressibleByNilLiteral {
 /// An `Activatable` element may allocate resources when `activate` is called. These
 /// resources may share information with any copies made from this element after `activate`
 /// was called.
-protocol Activatable {
+public protocol Activatable {
+    /// Activates the given element.
     mutating func activate()
```

Comment:
`Activatable` is public but luckily cannot be called by the user on property wrappers, because `handle()` is not mutating.